docs/messaging/nats-integration.md

```diff
@@ -2,110 +2,201 @@
 title: .NET Aspire NATS integration
 description: Learn how to use the .NET Aspire NATS integration to send logs and traces to a NATS Server.
 ms.topic: how-to
-ms.date: 08/12/2024
+ms.date: 10/09/2024
 ---
 
 # .NET Aspire NATS integration
 
-In this article, you learn how to use the .NET Aspire NATS integration to send logs and traces to a NATS Server. The integration supports persistent logs and traces across application restarts via configuration.
+[!INCLUDE [includes-hosting-and-client](../includes/includes-hosting-and-client.md)]
 
-## Prerequisites
+[NATS](https://nats.io) is a high-performance, secure, distributed messaging system. The .NET Aspire NATS integration enables you to connect to existing NATS instances, or create new instances from .NET with the [`docker.io/library/nats` container image](https://hub.docker.com/_/nats).
 
-- [Install the NATS server](https://docs.nats.io/running-a-nats-service/introduction/installation)
-- The URL to access the server.
+## Hosting integration
 
-## Get started
-
-To get started with the .NET Aspire NATS integration, install the [Aspire.NATS.Net](https://www.nuget.org/packages/Aspire.NATS.Net) NuGet package in the client-consuming project, i.e., the project for the application that uses the NATS client.
+The NATS hosting integration models a NATS server as the <xref:Aspire.Hosting.ApplicationModel.NatsServerResource> type. To access this type and APIs that allow you to add it to your app host, install the [📦 Aspire.Hosting.Nats](https://www.nuget.org/packages/Aspire.Hosting.Nats) NuGet package in the [app host](xref:aspire/app-host) project.
 
 ### [.NET CLI](#tab/dotnet-cli)
 
 ```dotnetcli
-dotnet add package Aspire.NATS.Net
+dotnet add package Aspire.Hosting.Nats
 ```
 
 ### [PackageReference](#tab/package-reference)
 
 ```xml
-<PackageReference Include="Aspire.NATS.Net"
+<PackageReference Include="Aspire.Hosting.Nats"
                   Version="[SelectVersion]" />
```

Comment:
Is there a specific reason you have `[SelectVersion]`? If you're just saying "the latest," use `*` instead.